### DISCUSSION TITLE
Arrow Keys does not show correctly - Extra Keys

### DISCUSSION THREAD

**Discussion by kafaa-dev** [termux/termux-app#3151] [Q&A] [answered]
![Screenshot_20221221-083937.jpg](https://user-images.githubusercontent.com/73120669/208800487-2067d48d-923b-4649-9741-26af2e672458.jpg)

**Comment by agnostic-apollo** [comment, marked as answer]
Add `extra-keys-style=none` to `~/.termux/termux.properties` and run `termux-reload-settings`.
